trolo/export/exporter.py

```diff
@@ -0,0 +1,155 @@
+from typing import Dict, Union, Optional, List, Tuple
+import torch
+import os
+import torch.nn as nn 
+import onnx
+from pathlib import Path
+from .base import BaseExporter
+from ..loaders import YAMLConfig
+from ..loaders.maps import get_model_config_path
+
+from ..utils.smart_defaults import infer_pretrained_model, infer_model_config_path
+from ..utils.logging.glob_logger import LOGGER
+
+class ModelExporter(BaseExporter):
```

Comment:
Do both BaseExport and ModelExporter need to exist? Maybe there's no need to have a base class in this case?